engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -1060,6 +1060,13 @@ - (void)flutterTextInputView:(FlutterTextInputView*)textInputView
   [self.platformPlugin showLookUpViewController:selectedText];
 }
 
+- (void)flutterTextInputView:(FlutterTextInputView*)textInputView
+    performPlatformChannelContextMenuCustomAction:(NSString*)customActionID
```

Comment:
nit: can you call it `performContextMenuCustomActionWithActionID:(NSString*)customActionID textInputClient:(int)client`?